frank-flow/src/frontend/src/app/shared/components/edit-dialog/edit-dialog.component.html

```diff
@@ -22,4 +22,8 @@ <h2 class="header" title="{{ oldFileName }}">
       type="text"
     />
   </div>
+  <div class="button-actions">
+    <app-button (click)="edit()">Confirm</app-button>
+    <app-button (click)="close()">Cancel</app-button>
```

Comment:
Closing the modal with 'X' or by clicking outside will still save the changes. This is very confusing. I would suggest changing that to cancel.